helion/_compiler/inductor_lowering.py

```diff
@@ -1396,6 +1417,12 @@ def _collect_multi_outputs(
 
     def run_node(self, n: Node) -> object:
         if n.op == "call_function":
+            # Skip codegen for nodes that can potentially be subtiled
+            if n.meta.get("epilogue_subtile", False):
+                arg = n.args[0]
+                assert isinstance(arg, Node)
+                return self.env[arg]
```

Comment:
Where are we codegenning this?  How do we ensure it is done in the correct order/place?